python/pyspark/worker.py

```diff
@@ -770,6 +770,7 @@ def verify_result(result):
                         message_parameters={
                             "type_name": type(result).__name__,
                             "value": str(result),
+                            "func": f.__name__,
```

Comment:
I guess we should do the same thing for `UDTF_RETURN_SCHEMA_MISMATCH` or other error classes if possible?

Reply:
Good point! I will update the other places.